_posts/2020-09-16-chross-browser-mobile-webapp.markdown

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: "Cross-Browser Mobile web apps (Snap Minis Part 2)"
+date: 2020-11-05 12:00:00
+categories: ["learning", "crossbrowser"]
+author: arieljuod
+---
+
+Following [the first part of this series](https://www.ombulabs.com/blog/learning/creating-a-snapchat-mini.html), I'll list a few interesting issues we found when making the apps work as similar as possible for Android's webview and iOS' webview.
+
+<!--more-->
+
+## Snap Minis
```

Comment:
I'm not sure that you need this title here since the whole article is about Minis, maybe you could remove this and then use ## for the subtitles in the rest of the article. I think that is better for SEO purposes?